src/app/movie-search/movie-search.component.ts

```diff
@@ -1,4 +1,5 @@
 import {Component, EventEmitter, OnInit, Output} from '@angular/core';
+import {Movie} from '../services/movie';
```

Comment:
This is not needed

Reply:
You are right this is superfluous. Removed